### PR TITLE
Type-hints added, method renamed

### PR DESCRIPTION
**Pull request by Quix0r**
Changes:
- added more type-hints
- added more documentation
- renamed `Item::visibileAcitivty()` to `Item::isVisibileAcitivty()` as this returns a boolean value
- reformatted some arrays

### CODE REVIEW

File: src/Content/BoundariesPager.php [hunk 1]
```diff
@@ -130,6 +130,9 @@ public function renderMinimal(int $itemCount): string
 		return Renderer::replaceMacros($tpl, ['pager' => $data]);
 	}
 
+	/**
+	 * Unsupported method, must be type-compatible
```
Comment (MrPetovan) [inline src/Content/BoundariesPager.php:134]: What does this mean?

**Review by MrPetovan — CHANGES_REQUESTED** [review]

**Comment by Quix0r** [comment]
I don't know for sure why the method is throwing an exception if called, commit e89e606b88 has added the thrown exception. So I just documented it, that it needs to be kept type-compatible to the parent `Pager::renderFull()` method.

File: src/Content/BoundariesPager.php (hunk 1 above)
Comment (Quix0r) [reply src/Content/BoundariesPager.php:134]: I don't know for sure why the method is throwing an exception if called, commit https://github.com/friendica/friendica/commit/e89e606b8827f6dc43f29db0965e27b12249d9dd has added the thrown exception. So I just documented it, that it needs to be kept type-compatible to the parent Pager::renderFull() method.

**Comment by MrPetovan** [comment]
I did that. It's a developer error where this method doesn't make sense in this sub-class compared to the parent.

**Review by MrPetovan — APPROVED** [review]

**Pull request merged**